values.yaml

```diff
@@ -139,7 +143,10 @@ global:
   loggingSidecar:
     enabled: false
     name: sidecar-log-consumer
-    image: quay.io/astronomer/ap-vector:0.40.1
+    images:
+      repository: quay.io/astronomer/ap-vector
+      tag: 0.40.1
+      pullPolicy: IfNotPresent
```

Comment:
let not add this pullPolicy in this PR this requires a code change

Reply:
ok, will revert this.